specification/azurefleet/AzureFleet.Management/fleet.tsp

```diff
@@ -211,6 +228,275 @@ union EvictionPolicy {
   Deallocate: "Deallocate",
 }
 
+@added(Versions.v2024_11_01)
+@doc("AdditionalCapabilities for VM.")
+model AdditionalCapabilities {
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "ultraSSDEnabled which does not follow camel casing"
+  @doc("""
+    The flag that enables or disables a capability to have one or more managed data disks with UltraSSD_LRS storage account type on the VM or VMSS.
+    Managed disks with storage account type UltraSSD_LRS can be added to a virtual machine or virtual machine scale set only if this property is enabled.
+    """)
+  ultraSSDEnabled?: boolean;
+
+  @doc("The flag that enables or disables hibernation capability on the VM.")
+  hibernationEnabled?: boolean;
+}
+
+#suppress "@azure-tools/typespec-azure-core/casing-style" "VMAttributeMinMaxInteger which does not follow camel casing"
+@added(Versions.v2024_11_01)
+@doc("While retrieving VMSizes from CRS, Min = 0 (uint.MinValue) if not specified, Max = 4294967295 (uint.MaxValue) if not specified. This allows to filter VMAttributes on all available VMSizes.")
+model VMAttributeMinMaxInteger {
+  @minValue(0)
+  @doc("Min VMSize from CRS, Min = 0 (uint.MinValue) if not specified.")
+  min?: int32;
+
+  @doc("Max VMSize from CRS, Max = 4294967295 (uint.MaxValue) if not specified.")
+  max?: int32;
+}
+
+#suppress "@azure-tools/typespec-azure-core/casing-style" "VMAttributeMinMaxDouble which does not follow camel casing"
+@added(Versions.v2024_11_01)
+@doc("VMAttributes using double values.")
+model VMAttributeMinMaxDouble {
+  @minValue(0.0)
+  @doc("Minimum value. default 0. Double.MinValue()")
+  min?: float64;
+
+  @doc("Maximum value. Double.MaxValue(1.7976931348623157E+308)")
+  max?: float64;
```

Comment:
You can add decorater @minValue(0.0) here as well as max value is always positive.

Reply:
dded @minValue(0.0)